web/client/src/components/soro-select.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+	<div class="material-select">
+		<select
+			class="material-select-text"
+			@change="emitChange"
+		>
+			<option
+				v-for="option in extendedOptions"
+				:key="option"
+				:value="option"
+				:selected="option === value"
+				:label="option ?? ''"
+			>
+				{{ option }}
+			</option>
+		</select>
+		<span class="material-select-highlight" />
+		<span class="material-select-bar" />
+		<label class="material-select-label">{{ label }}</label>
+	</div>
+</template>
```

Comment:
Is something wrong with the style?

Reply:
I seems to be that something is broken 👀